### DISCUSSION TITLE
Animated back to top button with CSS

### DISCUSSION THREAD

**Discussion by LITUATUI** [thegeeklab/hugo-geekblog#358] [Ideas]
The idea is to show a back to top button only after scrolling with CSS.

[This](https://www.freecodecamp.org/news/css-only-back-to-top-button/
) article shows some methods that only use CSS, no JS.

I especially like [this](https://codepen.io/t_afif/pen/ExwbZKQ/86f79c96520a0c864324db70a6b76c76) method with a fading button.

What do you guys think?

**Comment by xoxys** [comment]
See discussion at https://github.com/thegeeklab/hugo-geekdoc/discussions/578
